app/controllers/doorkeeper/token_info_controller.rb

```diff
@@ -6,7 +6,7 @@ def show
       if doorkeeper_token && doorkeeper_token.accessible?
         render json: doorkeeper_token, status: :ok
       else
-        error = OAuth::ErrorResponse.new(name: :invalid_request)
+        error = OAuth::InvalidTokenResponse.new
```

Comment:
I'm not entirely sure if `invalid_token` instead of `invalid_request` is valid here, since the `invalid_token` error is only mentioned in [RFC 6750](https://tools.ietf.org/html/rfc6750#section-3.1).

But it seems appropriate since the code above validates the token, also compare with a similar check in `lib/doorkeeper/rails/helpers.rb`

Reply:
Checked the document, agree